go/libkb/stream_classifier.go

```diff
@@ -127,10 +129,12 @@ type saltpackHeaderPrefix struct {
 
 func isSaltpackBinary(b []byte, sc *StreamClassification) bool {
 	if len(b) < 2 {
+		fmt.Printf("a\n")
```

Comment:
Yeah looks like it

On Wednesday, January 13, 2016, oconnor663 <notifications@github.com> wrote:

> In go/libkb/stream_classifier.go
> <https://github.com/keybase/client/pull/1727#discussion_r49685105>:
>
> > @@ -127,10 +129,12 @@ type saltpackHeaderPrefix struct {
> >
> >  func isSaltpackBinary(b []byte, sc *StreamClassification) bool {
> >  	if len(b) < 2 {
> > +		fmt.Printf("a\n")
>
> Is this test code that made it in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1727/files#r49685105>.
>